### ISSUE TITLE
Enable Replay cache for AFP3.x

### ISSUE THREAD

**Issue by anti-matter** [anti-matter/Windows_AFP_Server#1]
Maybe won't do this. Complicated and only needed in a case where a client crashes and wants to return to where they left off.

**Comment by NJRoadfan** [comment]
Replay cache is only really required for Time Machine support. How is the server storing FinderInfo and resource forks? NTFS has "official" ADS support for that. There are also extended attribute support in AFP 3.2 and later. I think Apple's SMB client has a way of storing these that should be followed.

**Comment by anti-matter** [comment]
Yes, I know the about the replay cache. FinderInfo and resource forks are stored in NTFS streams. I did not want to use other streams, documented or not, as I didn’t want to worry about conflicting formats. Extended attributes are already supported, but need to verify (it’s been a while since I worried about those).

UPDATE: I checked and extended attributes are indeed supported. They appear to work fine with MacOS X clients (which is the only OS that uses them). The attributes are stored in an NTFS stream for each attribute by name.

________________________________
